packages/nextjs/components/TrackedLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from "react";
+import { usePlausible } from "next-plausible";
+
+type TrackedLinkProps = {
+  id: string;
+  href: string;
+  className?: string;
+  children: ReactNode;
+};
+
+// Track external clicks on links using plausible
+export default function TrackedLink({ id, href, className, children }: TrackedLinkProps) {
+  const plausible = usePlausible();
+  console.log(id);
```

Comment:
maybe remove this console.log ?

Reply:
lol It seems that I missed that on the first BG commit, but fixed it later on: https://github.com/BuidlGuidl/buidlguidl.com/commit/31f9699cbbd9732a9bce99cfd7ebbccb3fd55621#diff-aa76b44f2c7ee26ccf78dcbd6091702e041178132b99f860141a8da237828d5a

Reply:
Yes! It was annoying hah

I checked BG website in local and realized it wasn't generating the annoying logs, but I guess I checked the wrong version of the source code in GH and thought the console.log was still in there.

Removed 🙌